app/services/weets.js

```diff
@@ -1,16 +1,30 @@
 const axios = require('axios');
```

Comment:
We can to create a helper to manage the requests to avoid a high coupling with axios package.